src/frontends/pytorch/src/op_table.cpp

```diff
@@ -280,6 +281,10 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"aten::leaky_relu", op::translate_1to1_match_2_inputs<opset10::PRelu>},
         {"aten::leaky_relu_", op::inplace_op<op::translate_1to1_match_2_inputs<opset10::PRelu>>},
         {"aten::len", op::translate_len},
+        // lift ops is torchscript specific ops responsible for tensors coping with guarantee of new memory allocation
```

Comment:
Does it mean that successive operations that consumes the result of lift* ops shouldn't use in-place semantics?